azure_arc_servers_jumpstart/bicep/main.bicep

```diff
@@ -1,6 +1,18 @@
 @description('Azure AD tenant id for your service principal')
 //param spnTenantId string
-param spnTenantId string = tenant().tenantId
+//param spnTenantId string = tenant().tenantId
+
+var spnTenantId = tenant().tenantId

```

Comment:
Let's keep it as a parameter and set the default value to `tenant().tenantId` to make the code reusable if we need to use it for a levelup

Reply:
ok

Reply:
done